app/src/main/java/com/nextcloud/talk/controllers/util/ControllerViewBindingDelegate.kt

```diff
@@ -25,13 +25,13 @@ import com.bluelinelabs.conductor.Controller
 import kotlin.properties.ReadOnlyProperty
 import kotlin.reflect.KProperty
 
-fun <T : ViewBinding> Controller.viewBinding(bindingFactory: (View) -> T) =
+fun <T : ViewBinding> Controller.viewBinding(bindingFactory: (View) -> T?) =
     ControllerViewBindingDelegate(this, bindingFactory)
 
 class ControllerViewBindingDelegate<T : ViewBinding>(
     controller: Controller,
-    private val viewBinder: (View) -> T
-) : ReadOnlyProperty<Controller, T>, LifecycleObserver {
+    private val viewBinder: (View) -> T?
+) : ReadOnlyProperty<Controller, T?>, LifecycleObserver {
```

Comment:
I think you can keep `bindingFactory` and `viewBinder` as returning `T` (not nullable). The only place it needs to be nullable is on the `getValue` and the `ReadOnlyProperty` definition.